crates/transaction-workloads-lib/src/move_workloads.rs

```diff
@@ -899,6 +920,9 @@ impl EntryPointTrait for EntryPoints {
             EntryPoints::DeserializeU256 => AutomaticArgs::None,
             EntryPoints::IncGlobalMilestoneAggV2 { .. } => AutomaticArgs::None,
             EntryPoints::CreateGlobalMilestoneAggV2 { .. } => AutomaticArgs::Signer,
+            EntryPoints::APTPermissionedTransfer | EntryPoints::APTTransfer => {
```

Comment:
nit: we can probably `|` with the above

Reply:
I would keep it as is because I think the matching statements are grouped by functionalities looking at the code above.